src/actions/recomputeReduxState.js

```diff
@@ -458,26 +461,26 @@ export const createStateFromQueryOrJSONs = ({
     entropy.maxYVal = entropyMaxYVal;
   }
 
-  /* potentially calculate frequency (or update it!)
-  this needs to come after the colorscale & tree is set */
-  if (JSONs && JSONs.frequencies) {
-    frequencies = {loaded: true, version: 1, ...processFrequenciesJSON(JSONs.frequencies, tree, controls)};
-  } else if (frequencies && frequencies.loaded) { /* oldState */
-    frequencies.version++;
-    frequencies.matrix = computeMatrixFromRawData(
-      frequencies.data,
-      frequencies.pivots,
-      tree.nodes,
-      tree.visibility,
-      controls.colorScale,
-      controls.colorBy
-    );
-  }
+  // if (JSONs && JSONs.frequencies) {
+  //   NOW A SEPERATE ACTION
+  // }
+  // else if (frequencies && frequencies.loaded) { /* oldState */
+  //   frequencies.version++;
+  //   frequencies.matrix = computeMatrixFromRawData(
+  //     frequencies.data,
+  //     frequencies.pivots,
+  //     tree.nodes,
+  //     tree.visibility,
+  //     controls.colorScale,
+  //     controls.colorBy
+  //   );
+  // }
```

Comment:
I'd recommend just removing this block of unused code rather than commenting it out. Chunks of commented out code are a bit of a "code smell", especially when using a version control system. They're rarely necessary and tend to just accumulate as clutter.